---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
redirect_from:
  - /software
---

{% include base_path %}


Selected projects:

- [*CHR*](https://github.com/msesia/chr): powerful and adaptive conformal prediction intervals for regression.
- [*Conformal p-values*](https://github.com/msesia/conditional-conformal-pvalues): software for computing calibration-conditional conformal p-values for outlier detection.
- [*KnockoffGWAS*](https://msesia.github.io/knockoffgwas/): GWAS method localizing causal variants across the genome, accounting for population structure and familial relatedness (aka KnockoffZoom v2).
- [*ARC*](https://github.com/msesia/arc): Python package for adaptive and reliable classification.
- [*KnockoffZoom*](https://msesia.github.io/knockoffzoom/): Flexible tool for the multi-resolution localization of causal variants across the genome.
- [*DeepKnockoffs*](https://github.com/msesia/DeepKnockoffs): Python package for approximate knockoffs and model-free variable selection.
- [*SNPknock*](https://msesia.github.io/snpknock/): R package for generating knockoffs of hidden Markov models and genetic data. 
- [*SNPknock for Python*](https://github.com/msesia/snpknock-python): Python package for generating knockoffs of hidden Markov models and genetic data. Note: this package is not up-to-date compared to the corresponding R package; the R package is currently recommended.
- [*Knockoffs*](https://web.stanford.edu/group/candes/knockoffs/): R and MATLAB packages for powerful and versatile controlled variable selection.

Other projects are on [GitHub](https://github.com/msesia).