---
title: "Controlling the false discovery rate in GWAS with population structure"
collection: publications
permalink: /publication/2020-knockoffgwas
excerpt: "A knockoff-based method for the genetic mapping of complex traits at multiple resolutions accounting for population structure, and a large-scale application to the UK Biobank data."
date: 2020-08-04
venue: 'pre-print'
paperurl: 'https://doi.org/10.1101/2020.08.04.236703'
citation: 'Sesia, Bates, Cand√®s, Marchini, and Sabatti (2020). &quot;Controlling the false discovery rate in GWAS with population structure.&quot; <i>bioRxiv</i>.'
---

**Abstract**

This paper proposes a novel statistical method to address population structure in genome-
wide association studies while controlling the false discovery rate, which overcomes some
limitations of existing approaches. Our solution accounts for linkage disequilibrium and
diverse ancestries by combining conditional testing via knockoffs with hidden Markov models
from state-of-the-art phasing methods. Furthermore, we account for familial relatedness by
describing the joint distribution of haplotypes sharing long identical-by-descent segments
with a generalized hidden Markov model. Extensive simulations affirm the validity of this
method, while applications to UK Biobank phenotypes yield many more discoveries compared
to BOLT-LMM, most of which are confirmed by the Japan Biobank and FinnGen data.

[Download paper here](http://msesia.github.io/files/knockoffgwas.pdf)

[Download supplementary material here](http://msesia.github.io/files/knockoffgwas-supp.pdf)
